### ISSUE TITLE
Create maui specific runtime branch

### ISSUE THREAD

**Issue by steveisok** [dotnet/runtime#61736]
In order to align correctly with upcoming MAUI previews, we need to create a separate runtime branch and publish out of it.

- [ ] Create branch off of release/6.0.  Suggestions on name of the branch are welcome (release/6.0-maui ?).
- [ ] Turn off stable branding
- [ ] Validate packages are building correctly
- [ ] Run official build

**Comment by steveisok** [comment]
@Anipik What are your thoughts on branding?

**Comment by mmitche** [comment]
release/6.0-maui sounds good to me.

**Comment by directhex** [comment]
So the existing version stuff on the release/6.0 branch reads

```
  <PropertyGroup>
    <!-- The .NET product branding version -->
    <ProductVersion>6.0.1</ProductVersion>
    <!-- File version numbers -->
    <MajorVersion>6</MajorVersion>
    <MinorVersion>0</MinorVersion>
    <PatchVersion>1</PatchVersion>
    <SdkBandVersion>6.0.100</SdkBandVersion>
    <PreReleaseVersionLabel>servicing</PreReleaseVersionLabel>
    <PreReleaseVersionIteration></PreReleaseVersionIteration>
  </PropertyGroup>
```

Do we just want to change PreReleaseVersionLabel and PreReleaseVersionIteration? And do we want something unique like `mauipreview` or are we gonna recycle the more generic `preview` text? I wouldn't want to use `maui` because that doesn't indicate that it's not stable. And I guess we need it to be alphabetically before `servicing`?

**Comment by ericstj** [comment]
cc @eerhardt @safern 

What type of changes do you intend to make in this branch?  Anything to libraries API or packages?

**Comment by eerhardt** [comment]
I'm a bit confused at having 2 "6.x" branches. Do we plan on actually "releasing" bits from this new branch? Or is it only to stage/develop changes that will eventually merge into the real `release/6.0` branch? And we only "release" bits from `release/6.0`?

**Comment by steveisok** [comment]
> What type of changes do you intend to make in this branch? Anything to libraries API or packages?

We should be making mostly bug fixes in this branch.  We're going to have to ultimately run any changes we make through tactics.

It's my understanding that the primary purpose of this branch is to release preview labeled artifacts aligned to MAUI previews.  The previews may come out at a different cadence than what comes out of release/6.0.  We're talking about the mono workload manifest and workload packs (e.g. runtime packs for Android, iOS/tvOS, and MacCatalyst).

**Comment by safern** [comment]
We did something similar on the past for blazor on 3.1, where we had release/3.1-blazor.

> And I guess we need it to be alphabetically before servicing

Maybe we could publish to a different feed that only maui uses? Or that doesn't work?

@steveisok do we need to publish OOB packages or only runtime and ref packs?

**Comment by steveisok** [comment]
> Maybe we could publish to a different feed that only maui uses? Or that doesn't work?

I think the preview labeling will give enough isolation where we don't need a separate private feed.  These preview nugets will make it to nuget.org whenever a public maui preview happens. 

> @steveisok do we need to publish OOB packages or only runtime and ref packs?

Good question.  I think we may need to ship OOB packages too as it's possible we'll need to fix something.

**Comment by ericstj** [comment]
If that's the case, then you need to have a plan to integrate back into servicing, as well as have a plan for what ships when you do so (every package, or only those you change).  We can't fork the versions of any asset that is shared between MAUI and the existing .NET 6 product.

**Comment by directhex** [comment]
```
/usr/local/share/dotnet/sdk/6.0.100/Sdks/NuGet.Build.Tasks.Pack/buildCrossTargeting/NuGet.Build.Tasks.Pack.targets(221,5): error NU5026: The file '/Users/directhex/Projects/runtime/artifacts/bin/System.IO.Pipelines/net461-Release/System.IO.Pipelines.dll' to be packed was not found on disk. [/Users/directhex/Projects/runtime/src/libraries/System.IO.Pipelines/src/System.IO.Pipelines.csproj]
```

🤨

**Comment by ericstj** [comment]
@directhex are you asking for help in diagnosing?  If so it's probably better to show more of your work, perhaps collaborate in a PR branch, or share logs describing what you are trying to do.  If I had to guess you're running pack while also doing a vertical build.  Our vertical builds only build specific configurations (because dev's don't want to build .NETFramework assemblies when building the net6), but packages will require more.  Those packages are created in the "allConfigurations" leg where we build all targets for the libraries.

**Comment by directhex** [comment]
Yeah I just reached that conclusion, more or less. We have a few places where behaviour changes based on explicit checks for `'$(PreReleaseVersionLabel)' == 'servicing'` in eng/packaging.targets

**Comment by directhex** [comment]
Okay, I created a base release/6.0-maui branch which is the same as release/6.0, and am doing the branding etc work in a PR for visibility in https://github.com/dotnet/runtime/pull/61792

I expect this to fail, it fails locally

**Comment by safern** [comment]
> Yeah I just reached that conclusion, more or less. We have a few places where behaviour changes based on explicit checks for '$(PreReleaseVersionLabel)' == 'servicing' in eng/packaging.targets

Yeah on the Maui branch I would expect to update that behavior to be conditioned based on the Maui prerelease version label. I'll look at your PR
